pulseaudio-control.bash

```diff
@@ -211,9 +211,16 @@ function nextSink() {
         pacmd move-sink-input "$i" "$newSink"
     done
 
+
```

Comment:
Last tiny change before I merge. Can you remove this extra newline so that the style is somewhat consistent? Thanks!

Reply:
Sure, no problem!